### PR TITLE
Feat/upgrade1020 misc 1

### PR DESCRIPTION
**Pull request by mustermeiszer**
# Description
Miscellaneous changes for the next release

## Changes and Descriptions
* `pallet-order-book` implements `trait TokenSwaps` without `min_buy_amount` checks
* `axelar-gateway-precompile` in `Altair` uses `EnsureRootOr<HalfOfCouncil>`
* provides a `Multilocation` based account conversion runtime API

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by mustermeiszer** [comment]
Bit wild kind of changes. But I did not wanted to wait for CI with 3 little PRs. Sorry...

**Comment by mustermeiszer** [comment]
Fixed lints and checked locally. We can merge this without CI going through. All other checks where green.

**Review by cdamian — APPROVED** [review]
Not so familiar with the order-book, the rest of the changes seem OK.

File: runtime/altair/src/evm.rs [hunk 1]
```diff
@@ -91,6 +90,6 @@ impl pallet_ethereum_transaction::Config for Runtime {
 }
 
 impl axelar_gateway_precompile::Config for Runtime {
-	type AdminOrigin = EnsureRoot<AccountId>;
+	type AdminOrigin = EnsureRootOr<HalfOfCouncil>;
```
Comment (wischli) [inline runtime/altair/src/evm.rs:93]: Out of curiosity: Do want to propose limiting to `Root` long-term?

**Review by wischli — APPROVED** [review]
LGTM! Thanks for the OrderBook fix 🚀

File: runtime/altair/src/evm.rs (hunk 1 above)
Comment (mustermeiszer) [reply runtime/altair/src/evm.rs:93]: I guess gov v2 will anyways make Council obsolete, right?

**Pull request merged**